idaes/generic_models/properties/core/eos/tests/test_ideal.py

```diff
@@ -192,6 +192,17 @@ def test_cv_mol_phase_comp(m):
                 pyunits.mol/pyunits.K))
 
 
+@pytest.mark.unit
+def test_heat_capacity_ratio_phase(m):
+
+    m.props[1].cp_mol_phase = Var(m.params.phase_list)
+    m.props[1].cv_mol_phase = Var(m.params.phase_list)
+
+    for p in m.params.phase_list:
+        assert str(Ideal.heat_capacity_ratio_phase(m.props[1], p)) == (
```

Comment:
Probably not for this PR, but there really should be a set of tests for EoSBase as well, which should include this.